### ISSUE TITLE
Overlapping data files

### ISSUE THREAD

**Issue by bpbond** [COMPASS-DOE/ghg-processor#5]
@wilsonsj100

```
Metadata row 75 Gcrew_WC_1 start = 2022-09-19 09:26:40 matched 601 data rows
	2022-09-19 09:26:40 to 2022-09-19 09:31:39
Metadata row 76 Gcrew_WC_2 start = 2022-09-19 09:39:10 matched 600 data rows
	2022-09-19 09:39:10 to 2022-09-19 09:44:09
Metadata row 77 Gcrew_WC_3 start = 2022-09-19 09:57:30 matched 600 data rows
	2022-09-19 09:57:30 to 2022-09-19 10:02:29
Metadata row 78 Gcrew_WC_4 start = 2022-09-19 10:11:40 matched 600 data rows
	2022-09-19 10:11:40 to 2022-09-19 10:16:39
Metadata row 79 Gcrew_WC_5 start = 2022-09-19 10:32:00 matched 600 data rows
	2022-09-19 10:32:00 to 2022-09-19 10:36:59
```

There are two files for 2022-09-19, with overlapping timestamps but different measurements:
* TG10-01087-2022-09-18T060000.data.txt (has 9/18 data but also 2022-09-19 timestamps from 08:52:19 to 11:19:02)
* TG10-01286-2022-09-19T060000.data.txt (2022-09-19 timestamps from 08:17:58 to 11:41:40)

Presumably one was measuring SMARTX or something that morning, and its data accidentally included here?

**Comment by wilsonsj100** [comment]
How can they have overlapping timestamps if they are separate days? are we not taking the date into account?

This could be a common issue, but we could suggest downloading licor data day by day?

**Comment by bpbond** [comment]
Sorry if that was unclear. Not separate days; if time stamps are correct, they appear to be two licors both operating the morning of 9/19.

**Comment by wilsonsj100** [comment]
Ohhhhh I see now! Yes, that must have accidentally got included. According to the field docs, looks like we used Louise for the GCReW wetland fluxes on the 19th so Gru may haver been used with the smart chamber or someone else's project during the same time. This is unusual any thoughts on how to handle it? i can redownload that data for the 18th without the 19th?

**Comment by bpbond** [comment]
Yay for good record-keeping. 

Rather than redownloading, can't we just remove TG10-01286-2022-09-19T060000.data.txt from the repository? (Or do I have it backwards...)

**Comment by bpbond** [comment]
...aaah, or should I just remove the 9/19 data from that file?

**Comment by wilsonsj100** [comment]
I think we need that datafile - I believe 01286 is Louise and that is the file we want for our data on the 19th.

**Comment by wilsonsj100** [comment]
We need to remove the 9-19 data from the other file that is 01087 (Gru) that is usually used, but wasn't on that day

**Comment by wilsonsj100** [comment]
PS - I just changed the field data sheet set up for this coming season to include a spot for: Licor time, Actual time, and Machine that way we should be able to put this in the metadata some how for next year and we should then be able to pull the auxillary data from aquatrolls and weather stations from the same time periods :)

**Comment by bpbond** [comment]
Thank you! Looks good now:
```
Metadata row 75 Gcrew_WC_1 start = 2022-09-19 09:26:40 matched 301 data rows
	TG10-01286 from 2022-09-19 09:26:40 to 2022-09-19 09:31:39
Metadata row 76 Gcrew_WC_2 start = 2022-09-19 09:39:10 matched 300 data rows
	TG10-01286 from 2022-09-19 09:39:10 to 2022-09-19 09:44:09
Metadata row 77 Gcrew_WC_3 start = 2022-09-19 09:57:30 matched 300 data rows
	TG10-01286 from 2022-09-19 09:57:30 to 2022-09-19 10:02:29
Metadata row 78 Gcrew_WC_4 start = 2022-09-19 10:11:40 matched 300 data rows
	TG10-01286 from 2022-09-19 10:11:40 to 2022-09-19 10:16:39
Metadata row 79 Gcrew_WC_5 start = 2022-09-19 10:32:00 matched 300 data rows
	TG10-01286 from 2022-09-19 10:32:00 to 2022-09-19 10:36:59
```

**Comment by wilsonsj100** [comment]
Yay! Thank you so much!

**Comment by bpbond** [comment]
Fixed in a39332a.
